site/assets/scss/_variables.scss

```diff
@@ -11,3 +11,4 @@ $bd-warning:       #f0ad4e;
 $bd-danger:        #d9534f;
 $dropdown-active-icon: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'><path fill='#292b2c' d='M2.3 6.73L.6 4.53c-.4-1.04.46-1.4 1.1-.8l1.1 1.4 3.4-3.8c.6-.63 1.6-.27 1.2.7l-4 4.6c-.43.5-.8.4-1.1.1z'/></svg>");
 $sidebar-collapse-icon: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='16' height='16' viewBox='0 0 16 16'><path fill='none' stroke='rgba(0,0,0,.5)' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M5 14l6-6-6-6'/></svg>");
+$zindex-sticky:    1080;
```

Comment:
I wouldn't use the same variable name than in our core to avoid confusion: might be something like `$bd-zindex-sticky` to scope it to Bootstrap Docs, or even `$bd-navbar-zindex` since it's meant to be used in the navbar.

Reply:
Noted :-) and I added a new variable named `$bd-navbar-zindex`.